From 99004704f9a3374a1eae6fee2924b34e1182148e Mon Sep 17 00:00:00 2001
From: "Dr. Greg" <greg@enjellic.com>
Date: Thu, 11 Feb 2021 18:46:56 -0600
Subject: [PATCH 28/50] Simplify description of Turing event implementation
 file.

The documentation for the Turing event implementation needs to be
updated further.  Until that process is complete replace the
description of the file implementing the functionality to
indicate that the file contains the implementation of the ability
to manage a Turing event domain.
---
 security/integrity/ima/ima_events.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/security/integrity/ima/ima_events.c b/security/integrity/ima/ima_events.c
index 09b48fac17fb..4044c4dfccd7 100644
--- a/security/integrity/ima/ima_events.c
+++ b/security/integrity/ima/ima_events.c
@@ -10,8 +10,7 @@
  * License.
  *
  * File: ima_events.c
- *	Implements generation of a a Turing event identity.
- *	Implements maintenance of the events list.
+ *	Implements management of a Turing event domain.
  */
 #include <linux/namei.h>
 #include <linux/seq_file.h>
-- 
2.31.1

